### ISSUE TITLE
[hotfix-v0.5] etcd container start is too noisy because of initialization status checks without any delay

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/etcd-druid#253]
**Describe the bug:**
While looking into etcd logs (to find out what is the reason for etcd restart) we noticed that etcd container start is too noisy/verbose with regards to logging.
etcd-druid@v0.5.2 is using `etcd-custom-image@v3.4.13-bootstrap`. In this version of etcd-custom-image, in the bootstrap script there is no sleep between the status checks. See https://github.com/gardener/etcd-custom-image/blob/e7bc3639f1509f1c80f199cdcdbf87190f5bf5c4/etcd_bootstrap_script.sh#L35-L55
It seems that this is supposed to be improved in the latest etcd-custom-image with https://github.com/gardener/etcd-custom-image/commit/81f9d549bddc14a58cf4c17877fbd5221ebfff35.

**Expected behavior:**
To have a reasonable delay between the intialization status checks.

**How To Reproduce (as minimally and precisely as possible):**
Check etcd startup logs and make sure that there are tons of log entries on etcd startup because of initialization status checks without any delay.

**Logs:**

```
{"log":"Length: 8 [text/plain]\n"}
{"log":"Saving to: 'status'\n"}
{"log":"\n"}
{"log":"     0K                                                       100% 14.3M=0s\n"}
{"log":"\n"}
{"log":"2021-11-03 12:26:59 (14.3 MB/s) - 'status' saved [8/8]\n"}
{"log":"\n"}
{"log":"--2021-11-03 12:26:59--  https://etcd-main-local:8080/initialization/status\n"}
{"log":"Resolving etcd-main-local (etcd-main-local)... 127.0.0.1\n"}
{"log":"Connecting to etcd-main-local (etcd-main-local)|127.0.0.1|:8080... connected.\n"}
{"log":"HTTP request sent, awaiting response... \n"}
{"log":"  HTTP/1.1 200 OK\n"}
{"log":"  Content-Security-Policy: default-src 'self'\n"}
{"log":"  Strict-Transport-Security: max-age=31536000\n"}
{"log":"  Date: Wed, 03 Nov 2021 12:26:59 GMT\n"}
{"log":"  Content-Length: 8\n"}
{"log":"  Content-Type: text/plain; charset=utf-8\n"}
{"log":"Length: 8 [text/plain]\n"}
{"log":"Saving to: 'status'\n"}
{"log":"\n"}
{"log":"     0K                                                       100% 18.3M=0s\n"}
{"log":"\n"}
{"log":"2021-11-03 12:26:59 (18.3 MB/s) - 'status' saved [8/8]\n"}
{"log":"\n"}
{"log":"--2021-11-03 12:26:59--  https://etcd-main-local:8080/initialization/status\n"}
{"log":"Resolving etcd-main-local (etcd-main-local)... 127.0.0.1\n"}
{"log":"Connecting to etcd-main-local (etcd-main-local)|127.0.0.1|:8080... connected.\n"}
{"log":"HTTP request sent, awaiting response... \n"}
{"log":"  HTTP/1.1 200 OK\n"}
{"log":"  Content-Security-Policy: default-src 'self'\n"}
{"log":"  Strict-Transport-Security: max-age=31536000\n"}
{"log":"  Date: Wed, 03 Nov 2021 12:26:59 GMT\n"}
{"log":"  Content-Length: 8\n"}
{"log":"  Content-Type: text/plain; charset=utf-8\n"}
{"log":"Length: 8 [text/plain]\n"}
{"log":"Saving to: 'status'\n"}
{"log":"\n"}
{"log":"     0K                                                       100% 13.0M=0s\n"}
{"log":"\n"}
{"log":"2021-11-03 12:26:59 (13.0 MB/s) - 'status' saved [8/8]\n"}
{"log":"\n"}
{"log":"--2021-11-03 12:26:59--  https://etcd-main-local:8080/initialization/status\n"}
{"log":"Resolving etcd-main-local (etcd-main-local)... 127.0.0.1\n"}
{"log":"Connecting to etcd-main-local (etcd-main-local)|127.0.0.1|:8080... connected.\n"}
{"log":"HTTP request sent, awaiting response... \n"}
{"log":"  HTTP/1.1 200 OK\n"}
{"log":"  Content-Security-Policy: default-src 'self'\n"}
{"log":"  Strict-Transport-Security: max-age=31536000\n"}
{"log":"  Date: Wed, 03 Nov 2021 12:26:59 GMT\n"}
{"log":"  Content-Length: 8\n"}
{"log":"  Content-Type: text/plain; charset=utf-8\n"}
{"log":"Length: 8 [text/plain]\n"}
{"log":"Saving to: 'status'\n"}
{"log":"\n"}
{"log":"     0K                                                       100% 13.2M=0s\n"}
{"log":"\n"}
{"log":"2021-11-03 12:26:59 (13.2 MB/s) - 'status' saved [8/8]\n"}
{"log":"\n"}

...
```

**Screenshots (if applicable):**
![Screenshot 2021-11-04 at 11 20 46](https://user-images.githubusercontent.com/9372594/140288716-538e82ed-eedb-485a-9a45-33bf1f63b6db.png)

From the screenshot you can see that there are 1000 log entries in loki for exactly 1 second - all logs are between 12:26:59 and 12:27:00. This almost makes impossible looking into etcd logs (having also in mind that the Shoot controlplane loki returns max 1000 logs).

**Environment (please complete the following information):**
 - Etcd version/commit ID :
 - Etcd-druid version : v0.5.2 -> `etcd-custom-image@v3.4.13-bootstrap`
 - Cloud Provider [All/AWS/GCS/ABS/Swift/OSS]:

**Comment by timuthy** [comment]
@shreyas-s-rao can you take over?

**Comment by shreyas-s-rao** [comment]
Hi @ialidzhikov. This issue of high log volume is fixed in the image `etcd-custom-image@v3.4.13-bootstrap-1`. This is included in etcd-druid v0.6.0 and above, but since we skipped etcd-druid@v0.6.0 on g/g, this hasn't yet reached the landscapes. There is a tentative druid release this week, in which this fix will be included and shipped to the gardener landscapes.

**Comment by ialidzhikov** [comment]
> Hi @ialidzhikov. This issue of high log volume is fixed in the image `etcd-custom-image@v3.4.13-bootstrap-1`. This is included in etcd-druid v0.6.0 and above, but since we skipped etcd-druid@v0.6.0 on g/g, this hasn't yet reached the landscapes. There is a tentative druid release this week, in which this fix will be included and shipped to the gardener landscapes.

Thanks for the reply @shreyas-s-rao . If there is already an image with the fix `etcd-custom-image@v3.4.13-bootstrap-1` then I would rather suggest to proceed with a patch release for etcd-druid (v0.5.3) that updates etcd-custom-image from v3.4.13-bootstrap to v3.4.13-bootstrap-1. I am not even sure why it was not done so far. We don't have to wait for a new feature release of etcd-druid to adopt the fix for this issue.

**Comment by ialidzhikov** [comment]
Oh, okay. I now recall such update of the etcd image will lead to rollout of the etcd Pods and will lead to downtimes for the kube-apiservers. Okay, let's wait for etcd-druid@v0.6.0 as you suggested...

**Comment by shreyas-s-rao** [comment]
Yes, we want to minimize the number of etcd restarts until we have multi-node etcd rolled out to the landscapes. Anyway, the fixed `etcd-custom-image@v3.4.13-bootstrap-1` should be available soon on our landscapes anyway, with the etcd-druid@v0.7.0 release

**Comment by timuthy** [comment]
> Okay, let's wait for etcd-druid@v0.6.0 as you suggested...

With that being said, can we close this issue @ialidzhikov?
